### PR TITLE
refactor: cleaning of old storage service variables

### PR DESCRIPTION
**Pull request by theus77**
|Q              |A  |
|---------------|---|
|Bug fix?       |N|
|New feature?   |N|	
|BC breaks?     |Y|	
|Deprecations?  |Y|	
|Fixed tickets  |N|

### CODE REVIEW

**Review by laurensmertens — DISMISSED** [review]

File: composer.json [hunk 1]
```diff
@@ -61,7 +61,7 @@
 		"phpunit/phpunit" : "^7.1",
 		"symfony/test-pack": "^1.0",
 		"squizlabs/php_codesniffer": "3.4",
-		"rector/rector": "dev-master"
+		"rector/rector": "^0.8"
```
Comment (dameert) [inline composer.json:64]: i fixed this to 0.8.56. As this is a dev tool I prefer it to not auto update on each build.

Comment (dameert) [reply composer.json:64]: (already on develop branch)

File: src/Service/FileService.php [hunk 2]
```diff
@@ -7,91 +7,34 @@
...
-
-    public function getStorageService(string $storageServiceId): ?StorageInterface
-    {
-        foreach ($this->storageManager->getAdapters() as $storageService) {
-            if ($storageService->__toString() == $storageServiceId) {
-                return $storageService;
-            }
-        };
-        return null;
+        $this->uploadMinimumNumberOfReplications = 10;
```
Comment (dameert) [inline src/Service/FileService.php:37]: This is not new code, but should we not have this in config?
```suggestion
        $this->uploadMinimumNumberOfReplications = 10;
```

**Review by dameert — APPROVED** [review]
Toto update branch with develop and accept the composer.json rector version of develop.

File: src/Service/FileService.php (hunk 2 above)
Comment (theus77) [reply src/Service/FileService.php:37]: I thinking of adding a `read-only` and a `to-skip` flag in the EMS_STORAGE and implement a isReadOnly and  isToSkip methods in the storage services?

**Pull request merged**